distribution/docs/src/main/jdocs/content/_configuring/managing-apps-from-command-console-contents.adoc

```diff
@@ -0,0 +1,37 @@
+:title: Managing Applications From the ${command-console}
+:type: configuringCommandConsole
+:status: published
+:summary: Managing Applications From the ${command-console}.
```

Comment:
They should be sentence case. Some of them reference specific modules/components, which I've capitalized. Going to go back through and make sure they are clear and consistent.